src/base/bittorrent/sessionimpl.cpp

```diff
@@ -2569,52 +2569,24 @@ bool SessionImpl::cancelDownloadMetadata(const TorrentID &id)
 
 void SessionImpl::increaseTorrentsQueuePos(const QList<TorrentID> &ids)
 {
-    using ElementType = std::pair<int, const TorrentImpl *>;
-    std::priority_queue<ElementType
```

Comment:
Using `priority_queue` seems to be overthinking for such cases where we need to sort entire list only once.